composer.json

```diff
@@ -21,6 +21,7 @@
     },
     "require": {
         "php": "^7.1 || ^8.0",
+        "ext-dom": "*",
```

Comment:
@Kern046 this needs to be moved to `suggest`, and then if it's not loaded, validation should be skipped.

Reply:
Should it also trigger a deprecation or not ? If not, that means that people could have XSD validation enabled with the boolean, but ineffective due to the absence of the DOM extension.

Reply:
Gave if some thought, and I think it should, and then we throw in 3.0, otherwise people are going to report issues that could have been prevented by installing that extension.